iac-extensions/arm/src/main/java/org/sonar/iac/arm/plugin/ArmExtension.java

```diff
@@ -23,10 +23,19 @@
 
 public class ArmExtension {
 
+  public static final String REPOSITORY_KEY = "azureresourcemanager";
+
   private ArmExtension() {
   }
 
   public static void define(Plugin.Context context) {
-    // empty for now
+    // TODO: uncomment once we have the other extensions available
```

Comment:
I think this can be uncommented now

Reply:
Not yet, unfortunately: If I do so, I get an error message because it is expecting to have an associated profile extension, and since it cannot find it, it is not able to instantiate the bean and so crash.
Should be good in the second subticket!